metricbeat/module/http/server/data.go

```diff
@@ -77,6 +77,7 @@ func (p *metricProcessor) Process(event server.Event) (common.MapStr, error) {
 		return nil, errors.New(fmt.Sprintf("Unsupported Content-Type: %s", contentType))
 	}
 
+	// QUESTION: What is the exact expected behaviour here if no list of paths is set
```

Comment:
@vjsamuel Could you comment here? It seems right now if I just run it the events end up under `http/http`. Not sure if that is expected.

Reply:
```
func defaultHttpServerConfig() HttpServerConfig {
	return HttpServerConfig{
		DefaultPath: PathConfig{
			Path:      "/",
			Namespace: "http",
		},
	}
}
```
it just exposes an API on `/` where you can send the data and the default namespace would be http

Reply:
Ok, at the moment the default is `http/http` so this is a bug I assume.